### PR TITLE
Adds script to run the deid with environment settings

### PR DESCRIPTION
**Pull request by ksdkalluri**
* Updates run_curation_pipeline.sh to use new deid_runner script

### CODE REVIEW

File: data_steward/deid_runner.sh [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    ;;
+  --)
+    shift
+    break
+    ;;
+  *) break ;;
+  esac
+done
+
+if [[ -z "${key_file}" ]] || [[ -z "${cdr_id}" ]] || [[ -z "${cdr_id}" ]]; then
```
Comment (mark-velez) [inline data_steward/deid_runner.sh]: @ksdkalluri cdr_id appears twice

File: data_steward/deid_runner.sh [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+cp -R "${GAE_SDK_APPENGINE}" "${VENV_LIB_GOOGLE}"
+cp -R "${GAE_SDK_NET}" "${VENV_LIB_GOOGLE}"
+
+export BIGQUERY_DATASET_ID="${cdr_deid}"
+export PYTHONPATH="${PYTHONPATH}:${DEID_DIR}:${DIR}"
+
+# Get Git version tag
+tag=$(git describe --abbrev=0 --tags)
+version=${tag}
```
Comment (mark-velez) [inline data_steward/deid_runner.sh]:
```suggestion
# Version is the most recent tag accessible from the current branch
version=$(git describe --abbrev=0 --tags)
```

Comment (ksdkalluri) [reply data_steward/deid_runner.sh]: added this change.

**Comment by ksdkalluri** [comment]
> @ksdkalluri this looks good! Please just address the two minor fixes.
> 
> I feel like `deid_runner.sh` should be collocated with `run_curation_pipeline.sh`. What do you think about moving `deid_runner.sh` and `run_deid.py` to `tools/`? It wouldn't necessarily have to be part of this PR. I could take it on if you think it's a good idea.

I have made these changes. Please review.

**Pull request merged**